src/http/routing.rs

```diff
@@ -80,412 +145,534 @@ macro_rules! routes {
                 }))
             }
 
+            /// This returns the common identifiers, which are used in rate limit buckets.
+            /// If you are interested in less common identifiers,
+            /// please match the variants you are interested in.
+            #[must_use]
+            #[allow(unused_variables)] // prevent compiler from complaining about unused variables
+            pub fn get_common_identifiers(&self) -> RatelimitCause {
```

Comment:
```suggestion
            pub fn cause(&self) -> RatelimitCause {
```

Reply:
This does not really return the "cause" of the rate limit but the parts of the path, which was rate limited.
The "cause" would be the deletion of a message or something like that.

Reply:
So the type should not be caused `RatelimitCause`, it should be called `RatelimitPathParts` or something else.